lib/devices/S7.ts

```diff
@@ -78,7 +78,7 @@ export class S7Connection extends DeviceConnection {
      */
     async open() {
         if (!this.#s7Conn.isConnected) {
-            this.#s7Conn.connect()
+            this.#s7Conn.once("connect", async () => {})
```

Comment:
@grigals this isn't doing anything. It's effectively registering a listener for the `connect` event and then running the supplied async function, which has nothing in it. What's the reason for removing the call to `connect()`?

Reply:
It is as per example. 
https://github.com/st-one-io/nodes7

I've tested that it does not crash the app anymore and waits until the is a valid connection.

Reply:
Removed as redundant due to autoconnect.